bindings/python/docs/source/quickstart.rst

```diff
@@ -76,7 +76,12 @@ Nested data (embedded documents) are also supported::
   from pymongoarrow.api import Schema
   schema = Schema({'_id': int, 'amount': float, 'account': { 'name': str, 'account_number': int}})
 
+Arrays (and embedded arrays) are also supported::
```

Comment:
I think this should be "nested arrays" as well.  I used "embedded document" because that is what it is called in the spec.